projects/igniteui-angular/schematics/ng-add/add-normalize.ts

```diff
@@ -14,47 +14,51 @@ export const scssImport =
 
 
 export const addResetCss = (workspace: workspaces.WorkspaceDefinition, host: Tree): boolean => {
-    const project = getDefaultProjectFromWorkspace(workspace);
+    const projects = getProjectsFromWorkspace(workspace);
+    let status = false;
     let addPackage;
-    const styleExts = ['scss', 'sass', 'css', 'less', 'styl'];
-    const styleExt = styleExts.find(ext => host.exists(path.posix.join(project.sourceRoot, `styles.${ext}`)));
-    if (!styleExt) {
-        return false;
-    }
-    const stylesFile = path.posix.join(project.sourceRoot, `styles.${styleExt}`);
-
-    switch (styleExt) {
-        case 'sass':
-        case 'scss':
-            let content = host.read(stylesFile).toString();
-            if (content.indexOf(`~minireset.css/minireset`) === -1) {
-                content = scssImport + content;
-                host.overwrite(stylesFile, content);
+    projects.forEach(project => {
+        const styleExts = ['scss', 'sass', 'css', 'less', 'styl'];
```

Comment:
Maybe we should move this outside the `forEach` as well so that we don't declare it on each pass.